Formula/libgetdata.rb

```diff
@@ -12,14 +12,14 @@ class Libgetdata < Formula
   end
 
   option "with-fortran", "Build Fortran 77 bindings"
-  option "with-perl", "Build Perl binding"
```

Comment:
IMO it's worth introducing the `:perl` dependency _where_ it's optional anyway. It makes sense to build bindings against the version of Perl actually first in the user's `$PATH`, providing that's a suitable version of Perl to meet the requirement.

The only area we need to be particularly wary is where it isn't `:optional`, given that that drags in the need to install our `perl` (as is the situation with `python` as well).